doc/Makefile

```diff
@@ -59,6 +59,7 @@ glm_reports:
 
 sym_links_datasets:
 	# copy datasets RST from nilearn package to doc folder
+	rm -fr modules/description
```

Comment:
Is this relevant here? Looks like a "spill" from other make file-related PRs?

Reply:
ha yeah I was getting some troubles generating the doc locally

will remove for now and readd in a separate PR if I can confirm the problem